app/controllers/meetings/publish/uittreksels.js

```diff
@@ -1,92 +1,75 @@
 import Controller from '@ember/controller';
-import {task} from "ember-concurrency";
-import {tracked} from "@glimmer/tracking";
+import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
 import { fetch } from 'fetch';
 import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default class MeetingsPublishUittrekselsController extends Controller {
-  @tracked uittreksels;
+  @service publish;
   @tracked documentToPrint;
   @tracked showPrintModal;
 
   @tracked errors;
 
-
   initialize() {
-    this.uittreksels = [];
     this.documentToPrint = null;
     this.showPrintModal = false;
-    this.initializeUittreksels.perform();
+    this.loadExtractsTask.perform();
   }
 
   get meeting() {
     return this.model;
   }
 
-  @task
-  * initializeUittreksels() {
-    yield this.fetchUittreksels.perform();
+  get extracts() {
+    return this.publish.treatmentExtracts;
```

Comment:
I don't think this is sorted

Reply:
It's just sorted in the service already, frontend sort